feat: add optional base_url kwarg to OpenAIModel (#243)
pydantic/pydantic-ai: docs/install.md

...
```

#### `base_url` argument

> To use another OpenAI compatible API, such as [OpenRouter](https://openrouter.ai), you can configure that via the [`base_url` argument][pydantic_ai.models.openai.OpenAIModel.__init__]:

Reviewer: ```suggestion
To use another OpenAI-compatible API, such as [OpenRouter](https://openrouter.ai), you can make use of the [`base_url` argument][pydantic_ai.models.openai.OpenAIModel.__init__]:
```
